### DISCUSSION TITLE
3ºinformática "NSL" PWD

### DISCUSSION THREAD

**Discussion by joaopsil** [profmessias/curriculo#7] [Comentários]
João Pedro de Souza Silva https://github.com/joaopsil/Curriculo

**Comment by gabrielvianamota** [comment]
3° 01 informática 
Gabriel Viana de Oliveira Mota: https://github.com/gabrielvianamota/curriculo

**Comment by miguellaguiar** [comment]
Miguel de Aguiar Freitas
3° INF 
https://github.com/miguellaguiar/curriculo

**Comment by julianaevangelista** [comment]
Juliana Evangelista Ferreira
3° 01 Informática
https://github.com/julianaevangelista/curriculo

**Comment by Felipewagmaker** [comment]
Felipe Wagmaker Almeida Bernardo
3º01 Informática
https://github.com/FenixOPW/Curriculo

**Comment by SAUKZx** [comment]
SAMUEL DA SILVA 3° TÉCNICO INFORMÁTICA
https://github.com/SAUKZx/Curriculo

**Comment by brianamorim200** [comment]
BRIAN AMORIM DA SILVA 3° TÉCNICO INFORMÁTICA
https://github.com/brianamorim200/Curriculo

**Comment by luisa-medina** [comment]
Luisa Costa Medina
https://github.com/luisa-medina/curriculo

**Comment by JhotaT** [comment]
Jhonatas Thomé dos Santos 3° informatica
https://github.com/JhotaT/curriculo

**Comment by Daveszz** [comment]
David Santos Oliveira 3° INF
https://github.com/davehub2006/curriculo
